Update README.md

kuldeep7688 opened the pull request:
Updated the Concept Map project description in the README file.

kuldeep7688 committed:
Update README.md

Updated the Concept Map project description in the README file.

fanurs reviewed (changes requested):
The section "Different aspects..." is not needed for this central repo. Just keep it to the individual repo's readme.

Also, I don't think the link for "Michael Plante" is working.
<img width="104" alt="image" src="https://user-images.githubusercontent.com/21100851/196518984-3c330f99-4a30-4630-8286-b476b33f976c.png">

kuldeep7688 committed:
Update README.md

Updated jonas and michael's github profiles in concept-map-extension readme section

kuldeep7688 committed:
Merge branch 'MSU-AI:main' into main

kuldeep7688 committed:
Update README.md

Removed the different aspects of the project from the concept-map-extension readme

fanurs reviewed (approved):
LGTM
